### DISCUSSION TITLE
关于拜年纪不带bvid的链接有没有什么方法处理一下

### DISCUSSION THREAD

**Discussion by TimG233** [Nemo2011/bilibili-api#623] [Q&A] [answered]
有bvid的拜年纪链接当然可以提取出来bvid然后正常用`video.py`里的方法去获取想要的信息。
但类似这种`https://www.bilibili.com/festival/2023bnj`链接依然是可以用的，但是没有任何id信息。请问这种链接怎么获取到视频aid/bvid?还是说无法获取到。

任何建议都可，感谢！

**Comment by z0z0r4** [comment]
你可以直接试试看用 `parse_link` 函数解析，但不清楚一年了有没有改动

**Reply by z0z0r4** [reply, marked as answer]
以及2023的信息是
aid | 565417552
bvid | "BV1zv4y117zo"

**Reply by TimG233** [reply]
OK！感谢

**Comment by Nemo2011** [comment]
那个页面可以抓 `__INITIAL_STATE__`

**Reply by z0z0r4** [reply]
没事了，去年偷懒了，我说咋解析不了

https://github.com/Nemo2011/bilibili-api/blob/11c33b16003f3111684421fb9ad4cbea5833ef18/bilibili_api/utils/parse_link.py#L627-L632
